packages/esm-ward-app/src/index.ts

```diff
@@ -39,8 +39,22 @@ export const wardPatientWorkspace = getAsyncLifecycle(
   options,
 );
 
+export const wardPatientNotesWorkspace = getAsyncLifecycle(
```

Comment:
```suggestion
// t("inpatientNotesWorkspaceTitle", "In-patient notes")
export const wardPatientNotesWorkspace = getAsyncLifecycle(
```

Reply:
Translation key for the workspace title needs to be added in the `en.json`

Reply:
This change is added @mogoodrich, Thanks